### ISSUE TITLE
Error with 1.8.1 “Error importing Bitcoin Core descriptor wallet”

### ISSUE THREAD

**Issue by pjv** [sparrowwallet/sparrow#1185]
After upgrading to 1.8.1 on starting sparrow (which connects to a remote pruned node), I get an error that says:

The connected node is pruned at 2023/11/27, but the wallet birthday for XXX - Postmix is set to 2023/10/19.

Downgrading back to 1.8.0 I get no such error starting up.

<img width="402" alt="SCR-20231129-htgy" src="https://github.com/sparrowwallet/sparrow/assets/327716/88526b48-3cd5-4ce1-aeeb-cf1f099eab76">

[macOS intel]

**Comment by pjv** [comment]
thanks @craigraw,

so change this?:

<img width="377" alt="SCR-20231129-jnys" src="https://github.com/sparrowwallet/sparrow/assets/327716/b86b0976-918c-4e56-9bf1-68605fc7e85e">

to ??? - i notice that all the other wallets I have in there seem to have that defaulted to 20.

**Comment by craigraw** [comment]
You can try reducing it to 20.

That said, I'd like to understand this better. Can you tell me
1. What is the result of running the command above? I'm looking for the `range` field for the receive descriptor, as in

```
{
      "desc": "wpkh(redacted/0/*)#9zjvaeq2",
      "timestamp": 1630402592,
      "active": false,
      "range": [
        0,
        5034
      ],
      "next": 1035
    },
```

You should redact the xpub as I have done above.

2. In Sparrow, what is the highest used receive address index in this wallet? You can find it by hovering over the last used address in the Addresses tab, or looking at the index in the Receive tab.

The highest used receive address index plus the gap limit should be well below the top of the range in Bitcoin Core.

**Comment by pjv** [comment]
````
    {
      "desc": "wpkh(redacted/0/*)#m97e06gx",
      "timestamp": 1697717565,
      "active": false,
      "range": [
        0,
        4000
      ],
      "next": 8,
      "next_index": 8
    },
````
If I understand what index I am looking for, it’s number 7 (when i hover over the last used receive address I see a tooltip with `../0/7`)

**Comment by craigraw** [comment]
Thanks for the info. It doesn't appear to be https://github.com/sparrowwallet/sparrow/commit/eecb90e9b277c35e7aa7b0f8484d454675b5ad62 that is causing this then. 

I'll consider this further to try understand what's going on.

**Comment by pjv** [comment]
OK. fwiw, I tried changing the gap limit in that advanced dialog in Sparrow from 40 to 20 and then ran v. 1.8.1 again and still get the same error on starting up. I’ll downgrade again back to 1.8.0 for now just because I don’t like seeing error messages 😄.

**Comment by craigraw** [comment]
Ok - I've found and fixed an issue in 6ee3755c.

Can you try the beta release at https://github.com/craigraw/beta/releases/tag/1.8.2-cormorant-range and report back?

**Comment by pjv** [comment]
That seems to have fixed it. No error on starting Sparrow with that beta.
